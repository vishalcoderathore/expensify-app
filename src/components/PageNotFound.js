import React from 'react';
import {Link} from 'react-router-dom';

const PageNotFound = () => (
    <div>
        404. Page Not Found. <Link to="/">Go Home</Link>
    </div>
);

export default PageNotFound;