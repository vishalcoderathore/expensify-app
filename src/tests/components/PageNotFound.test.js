import React from 'react';
import PageNotFound from '../../components/PageNotFound';
import { shallow } from 'enzyme';


test('should render not fond page correctly', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});