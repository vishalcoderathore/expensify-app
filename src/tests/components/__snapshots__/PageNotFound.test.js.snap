// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not fond page correctly 1`] = `
<div>
  404. Page Not Found. 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
